flutter-studio/src/io/flutter/project/FlutterProjectSystem.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.project;
+
+import com.android.tools.idea.projectsystem.AndroidModuleSystem;
+import com.android.tools.idea.projectsystem.AndroidProjectSystem;
+import com.android.tools.idea.projectsystem.ProjectSystemSyncManager;
+import com.android.tools.idea.projectsystem.gradle.GradleProjectSystem;
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.vfs.VirtualFile;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.nio.file.Path;
+
+public class FlutterProjectSystem implements AndroidProjectSystem {
+  @NotNull final private GradleProjectSystem gradleProjectSystem;
+
+  public FlutterProjectSystem(Project project) {
+    gradleProjectSystem = new GradleProjectSystem(project);
+  }
+
+  @Nullable
+  @Override
+  public VirtualFile getDefaultApkFile() {
+    return gradleProjectSystem.getDefaultApkFile(); // TODO find the flutter binary
```

Comment:
Do we need this todo:? If so, can you provide more context (for whoever comes through here net)?

Reply:
Removed.